vite-project-kabinet/src/Content/PaginatedList.jsx

```diff
@@ -14,13 +16,53 @@ export const PaginatedList = ({items, itemsPerPage}) => {
         }
     };
 
+    // Fetch funkce
+    const getPostsWithCategoryName = async () => {
```

Comment:
Nebereš teď v potaz stránkování. To je záměr?

Reply:
Na to som sa ťa chcela opýtať. Viem, že si mi to ukazoval na meetingu, ale mohol by si mi prosím ťa ešte raz krok po kroku napísať, že ako to má celé fungovať? Btw. to stránkovanie aktuálne funguje.